dev/scripts/sortPackageJson.js

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env node
+
+/*---------------------------------------------------------------------------------------------
```

Comment:
I'm not super familiar with shebangs but according to [this](https://alexewerlof.medium.com/node-shebang-e1d4b02f731d) it seems like shebangs must be on the first line. Since the kernel checks if the first two characters are `#!`.

> Node.js will happily ignore this as a comment only if it is the very first line of the file (it won’t work even if there’s an empty line or //comment line before it). Browsers ignore it too (Chrome 74+, FF 67+).